stubs/3.4/asyncio/queues.pyi

```diff
@@ -28,10 +29,10 @@ class Queue(Generic[T]):
     def empty(self) -> bool: ...
     def full(self) -> bool: ...
     @coroutine
-    def put(self, item: T) -> None: ...
+    def put(self, item: T) -> Future: ...
```

Comment:
Could the return type be more specific? If not, probably better to use `Future[Any]`.